docs/source/highlights_v0.1.md

```diff
@@ -1,8 +1,8 @@
-# Modules for public alpha
+# Modules in v0.1(public alpha)
```

Comment:
the version number should be v0.1.0, we need a white space before `(`

Reply:
updated in the latest commit.
Thanks.